content/pt-br/about.md

```diff
@@ -15,11 +15,11 @@ Nós fornecemos gratuitamente os certificados digitais necessários para habilit
 
 Os princípios chave da Let's Encrypt são: 
 
-* <strong>Gratuito:</strong> Qualquer pessoa que possui um nome de domínio pode usar a Let's Encrypt para obter um certificado confiável com custo zero.
```

Comment:
This part had to be changed after we decided that Let's Encrypt is a "female" noun in PT-BR